python/tvm/script/printer/__init__.py

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
BTW, please also add this folder in `./tests/scripts/task_mypy.sh` so that it will have mypy coverage

Reply:
Done